### DISCUSSION TITLE
Reach components from JavaScript

### DISCUSSION THREAD

**Discussion by jonaswikstrom** [ant-design-blazor/ant-design-blazor#3421] [Q&A]
Hello,

We have a service built in Blazor Server that uses ant-design-blazor. We are currently working on enhancing the user experience in case of a server disconnection. Our goal is to open a modal dialog to inform the user about the lost connection. However, since this functionality is primarily managed through interop with JavaScript functions, we need to access and invoke the Ant Design modal component using this approach. Is this feasible?

**Comment by ElderJames** [comment]
Thanks for contacting us @jonaswikstrom , the modal component can't be invoked from JS, so you need to add the html structure with ant-design classname.
